packages/react-tabs/src/components/Tabs/Tabs.test.tsx

```diff
@@ -26,6 +26,11 @@ describe('Tabs', () => {
     Component: Tabs,
     displayName: 'Tabs',
     skipAsPropTests: true,
+    testOptions: {
+      'consistent-callback-args': {
+        ignoreProps: ['onTabClick'],
+      },
+    },
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/136194055-13166ac6-2c9a-4a83-9421-3ff1f022cc46.png)

Before we tested only number or arguments and it passed, but now as we test types it fails ✅